exporter/opencensusexporter/testdata/config.yaml

```diff
@@ -0,0 +1,23 @@
+receivers:
+  examplereceiver:
+
+processors:
+  exampleprocessor:
+
+exporters:
+  opencensus:
+  opencensus/2:
+    enabled: true
+    endpoint: "1.2.3.4:1234"
+    compression: "on"
+    num-workers: 123
+    cert-pem-file: /var/lib/mycert.pem
+    headers:
+      header1: 234
```

Comment:
using `{` and `}` syntax can you double-check that you can have headers with `.`? it should work but let's double check it.

Reply:
If it doesn't no need to fix it now, but then let's open an issue to track it.

Reply:
Sorry, I didn't get that. Can you expand?

Reply:
if you use ```yaml
   headers: { "can you have a . here?":"F0000000-0000-0000-0000-000000000000" }

Reply:
I see what you mean now. You don't need {}, a quoted string can be a key in YAML. Let me add an example to the test.

Reply:
Added, please have a look.

Reply:
nice